### PR TITLE
added the ability to grab the beforeRender and fixed the data method …

### PR DESCRIPTION
**Pull request by iamjonbradley**
…to use this->request->data and not add to this->request->query

### CODE REVIEW

File: src/CakeCeption.php [hunk 1]
```diff
@@ -134,6 +134,24 @@ public function cookies($cookies)
...
 
 	/**
+	 * Apply session variables to the request
+	 * --
+	 * There will be no traces of the $_SESSION var in CGI
+	 *
+	 * @param array $session_vars
+	 * @return $this CakeCeption
+	 */
+	public function session($session_vars)
```
Comment (srph) [inline src/CakeCeption.php]: I find it weird that we're using a snake_case here.

Comment (rmrhz) [inline src/CakeCeption.php]: It's probably developer preference. I'll take note

**Comment by iamjonbradley** [comment]
i never understood why people don't camelcase params, but its just how they are doing it for some reason. sorry it took so long to reply, but i'll be submitting a merge request to fix it

**Comment by iamjonbradley** [comment]
i just submitted the pull request

**Pull request merged**